src/bugreporter/test/odemis_bugreporter_test.py

```diff
@@ -171,6 +171,8 @@ def test_compress_file(self):
         os.remove(self.bugreporter.zip_fn)
 
     def test_window(self):
+        # Note: UIActionSimulator.Text() doesn't seem to work on wxPython 4.0.7,
+        # but it works again on wxPython 4.1.
```

Comment:
Would it make sense to check the wx version here and do some logging, or do all our systems use 4.1?